cmd/snap-seccomp/versioninfo.go

```diff
@@ -65,3 +66,12 @@ func showVersionInfo() error {
 	fmt.Fprintln(os.Stdout, vi)
 	return nil
 }
+
+func GoSeccompFeatures() string {
+	features := "-"
+	// as more features are added, make this colon-separated
+	if actLogSupported() {
+		features = "bpf-actlog"
+	}
+	return features
```

Comment:
(nitpick^2) I know this is a bit of "YAGNI" but I would prepare for the ":" a bit more, something like (mostly because the other side of the code in versioninfo.go is already prepared for this):
```
--- a/cmd/snap-seccomp/versioninfo.go
+++ b/cmd/snap-seccomp/versioninfo.go
@@ -23,6 +23,7 @@ import (
        "crypto/sha256"
        "fmt"
        "os"
+       "strings"
 
        "github.com/mvo5/libseccomp-golang"
 
@@ -68,10 +69,14 @@ func showVersionInfo() error {
 }
 
 func GoSeccompFeatures() string {
-       features := "-"
+       var features []string
        // as more features are added, make this colon-separated
        if actLogSupported() {
-               features = "bpf-actlog"
+               features = append(features, "bpf-actlog")
        }
-       return features
+
+       if len(features) == 0 {
+               return "-"
+       }
+       return strings.Join(features, ":")
 }
```